src/resources/postcss/views/day/_event.pcss

```diff
@@ -28,6 +28,40 @@
 		}
 	}
 
+	.tribe-events-calendar-day__event--featured {
+		position: relative;
+
+		&:before {
+			background: var(--color-accent-primary);
+			content: '';
+			height: 100%;
+			left: var(--grid-gutter-small-half);
+			position: absolute;
+			width: 3px;
+		}
+
+		@media (--viewport-medium) {
+			&:before {
+				width: 0;
```

Comment:
We should be removing this by using `content: none`. But also, can we reuse this for the desktop instead of creating another pseudo element below? We're copying most of the styles from the `&:before` above.

Reply:
@paulmskim thanks, updated the `content`. With regards to the pseudo, we're defining them for two different elements to match the design.

Reply:
Great stuff. For the pseudo-element, I understand why you did this, what I mean is we can do it with one pseudo-element and position it to match the design rather than with two pseudo-elements and copied styles.

Reply:
https://www.loom.com/share/856372c4fd5e49fd9bd78cab1b3c4a32

Reply:
So if you set `left: calc(var(--grid-width-1-of-9) - var(--grid-gutter-half));` on the article pseudo-selector for `--viewport-medium` and remove the `__event-details` pseudo-selector, you should get the same effect as the design but using the same pseudo-selector. Let's hop on a call if it doesn't work out.

Reply:
Ok, cool, thanks!